src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -47,16 +45,15 @@ def __exit__(self, *args):
 
 class JSONSerializer(Serializer):
     """
-    JsonSerializer serializes objects in the Confluent Schema Registry binary
-    format for JSON.
+    Serializer that outputs JSON formatted data with Confluent Schema Registry framing.
 
-    JsonSerializer configuration properties:
+    Configuration properties:
 
     +---------------------------+----------+--------------------------------------------------+
     | Property Name             | Type     | Description                                      |
     +===========================+==========+==================================================+
-    |                           |          | Registers schemas automatically if not           |
-    | ``auto.register.schemas`` | bool     | previously associated with a particular subject. |
+    |                           |          | If True, automatically register unrecognized     |
```

Comment:
I think the former wording was clearer. Now we have to wonder what an unrecognized schema is.

Reply:
yes and yes.